velero-plugin-for-microsoft-azure/volume_snapshotter.go

```
@@ -413,12 +420,19 @@ func (b *VolumeSnapshotter) SetVolumeID(unstructuredPV runtime.Unstructured, vol
 		return nil, errors.WithStack(err)
 	}
 
-	if pv.Spec.AzureDisk == nil {
-		return nil, errors.New("spec.azureDisk not found")
-	}
+	if pv.Spec.CSI != nil && pv.Spec.CSI.Driver == "disk.csi.azure.com" {
+		pv.Spec.CSI.VolumeHandle = getComputeResourceName(b.disksSubscription, b.disksResourceGroup, disksResource, volumeID)
```

Should return an error if the driver is not "disk.csi.azure.com"?